exchange/auction.go

```diff
@@ -73,11 +73,11 @@ func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client,
 	toCache := make([]prebid_cache_client.Cacheable, 0, expectNumBids+expectNumVast)
 	expByImp := make(map[string]int64)
 	competitiveExclusion := false
-	var hbCacheID string
+	var HbCacheID string
```

Comment:
Why capitalize `HbCacheID`? Normally that would indicate that `HbCacheID` is being exported, but that is not possible in this scope.

Reply:
Ah! Good catch @hhhjort. This wasn't on purpose but an effect of rename all :p I will change it back. Thanks for pointing it out.

Reply:
Updated!